docs/docker/README.md

```diff
@@ -31,8 +33,18 @@ $ docker pull mcr.microsoft.com/playwright:bionic
 
 ### Run the image
 
+By default, the Docker image will use the `root` user to run the browsers. This will disable the Chromium sandbox which is not available with root. If you run trusted code (e.g. End-to-end tests), then the root user is in most cases fine. For web scraping or crawling, we recommend to create a separate user inside the Docker container and use the seccomp profile.
```

Comment:
> If you run trusted code (e.g. End-to-end tests), then the root user is in most cases fine.

Let's rephrase this to not promote running under root. Maybe something like 'If you run trusted code (e.g. End-to-end tests) and want to avoid the hassle of managing separate user then the root user may be fine.'